[BUG]: Warning:  Failed to translate locale

**Describe the bug**
In a collection type when I want to `Translate from another locale` I get the following error:

```
TypeError: Cannot read properties of undefined (reading 'post')
    at mn (main.500a0f16.js:1258:1010)
    at onClick (main.500a0f16.js:3613:251)
    at Object.Y (main.500a0f16.js:101:9347)
    at Je (main.500a0f16.js:101:9507)
    at He (main.500a0f16.js:101:9570)
    at Fi (main.500a0f16.js:101:29084)
    at Ur (main.500a0f16.js:101:29545)
    at main.500a0f16.js:101:34949
    at Gt (main.500a0f16.js:105:19082)
    at sn (main.500a0f16.js:101:8502)
```

Attempting the same from the DeepL settings menu works without any issues. 

**To Reproduce**
Steps to reproduce the behavior:
1. Go to '...'
2. Click on '....'
3. Scroll down to '....'
4. See error

**Expected behavior**
A clear and concise description of what you expected to happen.

**Screenshots**
If applicable, add screenshots to help explain your problem.

**System (please complete the following information):**
 - OS: Mac
 - Browser brave
 - Dependencies:

```js
  "dependencies": {
    "@ckeditor/strapi-plugin-ckeditor": "^0.0.7",
    "@strapi/plugin-documentation": "^4.7.0",
    "@strapi/plugin-i18n": "^4.7.0",
    "@strapi/plugin-open-ai": "^1.0.1",
    "@strapi/plugin-users-permissions": "4.7.0",
    "@strapi/provider-upload-aws-s3": "^4.7.0",
    "@strapi/strapi": "4.7.0",
    "date-fns": "^2.29.3",
    "framer-motion": "^10.0.1",
    "pg": "^8.9.0",
    "pg-connection-string": "^2.5.0",
    "react-select": "^5.7.0",
    "sharp": "^0.31.3",
    "strapi-plugin-config-sync": "^1.1.1",
    "strapi-plugin-deepl": "^0.4.2",
    "strapi-plugin-populate-deep": "^2.0.0",
    "strapi-plugin-sitemap": "^2.0.9"
  },
```

**Plugin config**
Please add your plugin config here, e.g.
```js
module.exports = ({ env }) => ({
  // ...
  'config-sync': {
    enabled: true,
    config: {
      syncDir: 'config/sync/',
      minify: false,
      importOnBootstrap: false,
      customTypes: [],
      excludedTypes: [],
      excludedConfig: ['core-store.plugin_users-permissions_grant'],
    },
  },
  deepl: {
    enabled: true,
  },
  upload: {
    config: {
      provider: 'aws-s3',
      providerOptions: {
        accessKeyId: env('AWS_ACCESS_KEY_ID'),
        secretAccessKey: env('AWS_ACCESS_SECRET'),
        region: env('AWS_REGION'),
        params: {
          Bucket: env('AWS_BUCKET'),
        },
      },
    },
  },
  'open-ai': {
    enabled: true,
    config: {
      API_TOKEN: env('OPENAI_API_KEY'),
    },
  },
  'strapi-plugin-populate-deep': {
    config: {
      defaultDepth: 3, // Default is 5
    },
  },
})
```

**Additional context**
I tried deleting the whole posts collection, but it seems that somehow the issue persists. How could I further debug this to understand where exactly the translation is failing?

Thanks a ton for your help and developing this amazing plugin in the first place 💪

@sargreal sorry, super stupid mistake. I followed the upgrade documentation and now have the following:

`plugin.js`
```js
module.exports = ({ env }) => ({
  // ...
  'config-sync': {
    enabled: true,
    config: {
      syncDir: 'config/sync/',
      minify: false,
      importOnBootstrap: false,
      customTypes: [],
      excludedTypes: [],
      excludedConfig: ['core-store.plugin_users-permissions_grant'],
    },
  },
  translate: {
    enabled: true,
    config: {
      // Add the name of your provider here (for example 'deepl' for strapi-provider-translate-deepl or the full package name)
      provider: 'deepl',
      providerOptions: {
        // your API key - required and wil cause errors if not provided
        apiKey: env('DEEPL_API_KEY'),
        // use custom api url - optional
        apiUrl: 'https://api-free.deepl.com',
      },
      // Which field types are translated (default string, text, richtext, components and dynamiczones)
      // Either string or object with type and format
      // Possible formats: plain, markdown, html (default plain)
      translatedFieldTypes: [
        'string',
        { type: 'text', format: 'plain' },
        { type: 'richtext', format: 'html' },
        'component',
        'dynamiczone',
      ],
      // If relations should be translated (default true)
      translateRelations: true,
    },
  },
  upload: {
    config: {
      provider: 'aws-s3',
      providerOptions: {
        accessKeyId: env('AWS_ACCESS_KEY_ID'),
        secretAccessKey: env('AWS_ACCESS_SECRET'),
        region: env('AWS_REGION'),
        params: {
          Bucket: env('AWS_BUCKET'),
        },
      },
    },
  },
  'open-ai': {
    enabled: true,
    config: {
      API_TOKEN: env('OPENAI_API_KEY'),
    },
  },
  'strapi-plugin-populate-deep': {
    config: {
      defaultDepth: 3, // Default is 5
    },
  },
})
```

`package.json`
```js
  "dependencies": {
    "@ckeditor/strapi-plugin-ckeditor": "^0.0.7",
    "@strapi/plugin-documentation": "^4.7.0",
    "@strapi/plugin-i18n": "^4.7.0",
    "@strapi/plugin-open-ai": "^1.0.1",
    "@strapi/plugin-users-permissions": "4.7.0",
    "@strapi/provider-upload-aws-s3": "^4.7.0",
    "@strapi/strapi": "4.7.0",
    "date-fns": "^2.29.3",
    "framer-motion": "^10.0.1",
    "pg": "^8.9.0",
    "pg-connection-string": "^2.5.0",
    "react-select": "^5.7.0",
    "sharp": "^0.31.3",
    "strapi-plugin-config-sync": "^1.1.1",
    "strapi-plugin-populate-deep": "^2.0.0",
    "strapi-plugin-sitemap": "^2.0.9",
    "strapi-plugin-translate": "^1.1.0"
  },
```
I am running now into a similar issue as mentioned here: #96.

Steps:

1. `npm run build` (works without issues)
2. `npm run dev`:

```bash
[2023-03-01 12:20:26.413] debug: ⛔️ Server wasn't able to start properly.
[2023-03-01 12:20:26.416] error: Could not load translate provider "deepl".
Error: Could not load translate provider "deepl".
    at createProvider (/Users/felixvemmer/Developer/cryptoticker/node_modules/strapi-plugin-translate/server/bootstrap.js:29:13)
    at Object.module.exports [as bootstrap] (/Users/felixvemmer/Developer/cryptoticker/node_modules/strapi-plugin-translate/server/bootstrap.js:41:41)
    at Object.bootstrap (/Users/felixvemmer/Developer/cryptoticker/node_modules/@strapi/strapi/lib/core/domain/module/index.js:40:47)
    at Object.bootstrap (/Users/felixvemmer/Developer/cryptoticker/node_modules/@strapi/strapi/lib/core/registries/modules.js:28:19)
    at processTicksAndRejections (node:internal/process/task_queues:96:5)
    at async Strapi.runLifecyclesFunctions (/Users/felixvemmer/Developer/cryptoticker/node_modules/@strapi/strapi/lib/Strapi.js:537:5)
    at async Strapi.bootstrap (/Users/felixvemmer/Developer/cryptoticker/node_modules/@strapi/strapi/lib/Strapi.js:475:5)
    at async Strapi.load (/Users/felixvemmer/Developer/cryptoticker/node_modules/@strapi/strapi/lib/Strapi.js:484:5)
    at async Strapi.start (/Users/felixvemmer/Developer/cryptoticker/node_modules/@strapi/strapi/lib/Strapi.js:215:9)
npm ERR! Lifecycle script `dev` failed with error: 
npm ERR! Error: command failed 
npm ERR!   in workspace: @cryptoticker/strapi@0.1.0 
npm ERR!   at location: /Users/felixvemmer/Developer/cryptoticker/apps/strapi 
```
The API key is set in `.env`. Do you think it's more related to a faulty config, or that something is messed up in the `schema json`.

🙏

--- a/plugin/admin/src/components/CMEditViewTranslateLocale/index.js
+++ b/plugin/admin/src/components/CMEditViewTranslateLocale/index.js
@@ -45,11 +45,11 @@ import {
   useNotification,
   useQueryParams,
   CheckPermissions,
+  request,
 } from '@strapi/helper-plugin'
 import { generateOptions } from '@strapi/plugin-i18n/admin/src/components/CMEditViewInjectedComponents/CMEditViewCopyLocale/utils'
 import useContentTypePermissions from '@strapi/plugin-i18n/admin/src/hooks/useContentTypePermissions'
 import selectI18NLocales from '@strapi/plugin-i18n/admin/src/selectors/selectI18nLocales'
-import { axiosInstance } from '@strapi/plugin-i18n/admin/src/utils'
 import _ from 'lodash'
 import { getTrad } from '../../utils'
 import permissions from '../../permissions'
@@ -153,11 +153,14 @@ const Content = ({
       const { locale: sourceLocale } = localizations.find(
         ({ id }) => id == value
       )
-      const { data: translatedData } = await axiosInstance.post(translateURL, {
-        id: value,
-        sourceLocale,
-        targetLocale: currentLocale,
-        contentTypeUid: slug,
+      const translatedData = await request(translateURL, {
+        method: 'POST',
+        body: {
+          id: value,
+          sourceLocale,
+          targetLocale: currentLocale,
+          contentTypeUid: slug,
+        },
       })
 
       const parsedData = parseRelations(translatedData, allLayoutData)
@@ -247,7 +250,7 @@ const Content = ({
       {isOpen && (
         <Dialog onClose={handleToggle} title="Confirmation" isOpen={isOpen}>
           <DialogBody icon={<ExclamationMarkCircle />}>
-            <Stack size={2}>
+            <Stack spacing={2}>
               <Flex justifyContent="center">
                 <CenteredTypography id="confirm-description">
                   {formatMessage({

--- a/plugin/package.json
+++ b/plugin/package.json
@@ -12,6 +12,7 @@
     "lint": "eslint ."
   },
   "dependencies": {
+    "@strapi/helper-plugin": "^4.1.4",
     "axios": "^1.2.2",
     "bottleneck": "^2.19.5",
     "jsdom": "^21.0.0",

